schema/crdb/README.adoc

```diff
@@ -7,28 +7,31 @@ This directory describes the schema(s) used by CockroachDB.
 
 We use the following conventions:
 
-* `schema/crdb/VERSION/up.sql`: The necessary idempotent migrations to
-  transition from the previous version of CockroachDB to this version. These
```

Comment:
I was particularly confused by "These migrations will always be placed within one transaction per file."  I wasn't sure what "migration" meant (one statement?  all the statements in one file?  all of the `up*.sql` files for one version?).  So I avoided this term and used "statements" and "all of the statements in a given file" instead.

I'm also not positive I interpreted this right.  I interpreted this to mean "the framework will take all of the statements in one file and execute them in one transaction."  It may have meant "you are responsible for putting exactly one transaction in the file" (i.e., it should have `BEGIN` and `END`, etc.)  I think this was once the case, but it's empirically not the case right now.

Reply:
> I'm also not positive I interpreted this right. I interpreted this to mean "the framework will take all of the statements in one file and execute them in one transaction."

This is correct -- the framework also tracks some version information within this transaction, to make it safe for multiple Nexus instances to perform these updates concurrently.

> It may have meant "you are responsible for putting exactly one transaction in the file" (i.e., it should have BEGIN and END, etc.) I think this was once the case, but it's empirically not the case right now.

Yeah, this **was** the case, but it is not the case any longer.